src/app/components/cards/PostFull.jsx

```diff
@@ -170,6 +171,18 @@ class PostFull extends React.Component {
         );
     }
 
+    redditShare(e) {
+        serverApiRecordEvent('RedditShare', this.share_params.link);
+        e.preventDefault();
+        const s = this.share_params;
+        const q =
+            'title=' +
+            encodeURIComponent(s.title) +
+            '&url=' +
+            encodeURIComponent(s.url);
+        window.location = '//www.reddit.com/submit?' + q;
```

Comment:
Don't we what `https://` here?

Reply:
Also, the other share options are using `window.open`, perhaps that's preferable.

Reply:
Good to be explicit, though should use `https` since steemit.com will always be